### PR TITLE
remixd as a module

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: src/app/files/remixd-handle.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  .dialog {
+    display: flex;
+    flex-direction: column;
+  }
+  .dialogParagraph {
+    ${styles.infoTextBox}
+    margin-bottom: 2em;
+    word-break: break-word;
+  }
+`
```
Comment (LianaHus) [inline src/app/files/remixd-handle.js]: 2 empty lines here

File: src/app/files/remixd-handle.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    this.locahostProvider.close((error) => {
+      if (error) console.log(error)
+    })
+  }
+
+  activate () {
+    this.connectToLocalhost()
+  }
+
+   /**
```
Comment (LianaHus) [inline src/app/files/remixd-handle.js]: you need to shift comment to the left :)

File: src/app/files/remixd-handle.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        { label: 'Connect',
+          fn: () => {
+            this.locahostProvider.init((error) => {
+              if (error) {
+                console.log(error)
+              } else {
+                this.fileSystemExplorer.ensureRoot()
+              }
+            })
+          }})
```
Comment (LianaHus) [inline src/app/files/remixd-handle.js]: closing brace and parenthesis are  on a wrong line here

File: src/app/files/remixd-handle.js (hunk 1 above)
Comment (yann300) [reply src/app/files/remixd-handle.js]: you've missed "`" ;)

**Pull request merged**